### PR TITLE
Merging Develop into Master

### PR DESCRIPTION
**Pull request by B3rry**

### CODE REVIEW

**Comment by jsachs** [comment]
Can one of the admins verify this patch?

File: config.js.docker [hunk 1]
```diff
@@ -13,6 +13,14 @@ config.text = {
 
 // client secret
 config.secret = "testsecret";
+config.jwtSecret = "0a6b944d-d2fb-46fc-a85e-0295c986cd9f";
+
+// Auth Service Connections
+if (process.env.NODE_ENV === "production") {
+    config.authServiceAPI = "http://api-auth-service-lb-2033098035.us-west-2.elb.amazonaws.com/api/v0"
```
Comment (jsachs) [inline config.js.docker]: @B3rry can we pass the LB URLs as env vars?

**Comment by jsachs** [comment]
@B3rry or @nickmccally same question as above, make sure to do a sweep of the repo and replace any URIs or other AWS-identifying information with env vars.

**Comment by B3rry** [comment]
Closing, won't merge as discussed during standup

**Pull request closed without merging**